src/gmail.js

```diff
@@ -1089,6 +1142,16 @@ var Gmail = function(localJQuery) {
                 }
             }
         }
+
+        if (threads[0]) {
+            actionType = api.tools.check_event_type(threads[0]);
+        }
+
+        if (actionType) {
+            const new_thread_ids = threads.map(thread => thread[1]);
```

Comment:
This if-statement will only ever run if the previous if-statement has run, so it might as well be inside that if-block instead of in its own.

And if you do that, you can also make the `actionType` a block-level variable making the overall structure of the function cleaner.